src/stactools/core/utils/antimeridian.py

```diff
@@ -149,3 +191,36 @@ def normalize(polygon: Polygon) -> Optional[Polygon]:
         return shapely.affinity.translate(polygon, xoff=+360)
     else:
         return polygon
+
+
+def normalize_multipolygon(multi_polygon: MultiPolygon) -> Optional[MultiPolygon]:
+    """'Normalizes' a WGS84 lat/lon multi polygon, or returns None if no changes were made.
+
+    This converts the multi-polygon's x coordinates to all be the same sign, even if
+    the polygon crosses the antimeridian.
```

Comment:
Maybe update the explanation to make the outcome clear. Something like:
>For each polygon in the multi-polygon, this converts the x coordinates to all be the same sign, even if the polygon crosses the antimeridian. Although the x coordinate sign within each polygon will be made the same, the sign may differ between polygons depending on their position relative to the antimeridian.

Reply:
Updated w/ your text.